bfdd/bfdd_vty.c

```diff
@@ -685,6 +685,9 @@ static int bfd_configure_peer(struct bfd_peer_cfg *bpc, bool mhop,
 			snprintf(ebuf, ebuflen, "vrf name too long");
 			return -1;
 		}
+	} else {
+		bpc->bpc_has_vrfname = true;
```

Comment:
Given this change -- is there any instance when `bpc_has_vrfname` will be false? If not, is this flag needed any longer?

Reply:
Actually, this is used only when using external bfd daemon (ptm) . see file bfd/config.c for more information.
allocating structure defaults the value to 0 ( false). that is why it can make thinknig that always true value is used.

I think that simplification can be done in a separate pr ( provided that in bfd/config.c, we provision it with appropriate vrf).

Reply:
okay -- let's raise a pr for this and for is-is (if it needs to be) -- I'll merge this one.